systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -762,6 +864,18 @@ void testRestoreOffsetsInConsumerGroup() {
     }
 
     @ParallelNamespaceTest
+    @TestDoc(
+        description = @Desc("Checks MM2 connector state transitions and offset management, including error handling."),
+        steps = {
+            @Step(value = "Deploy clusters and MM2 with wrong config to force connector failure.", expected = "MM2 shows NotReady with error."),
+            @Step(value = "Fix config, MM2 becomes Ready.", expected = "Connectors start working after fix."),
```

Comment:
maybe more details about the config? What is set to simulate NotReady and how it is fixed?